Update ssf.txt

PL1-Arcade opened the pull request:
Minor update to Polygonet Commanders and two example screencaps for README

PL1-Arcade commented:
Not sure if the captions are properly worded.
Can redo/adjust them as needed.
Would it be better to do screencaps showing the method currently mentioned in the README?

Jakobud commented:
Go ahead and remove the images. I'm working on improving the readme. I can merge in the ssf.txt changes though. Also working on setting default values as you discussed.

PL1-Arcade commented:
I **think** I deleted the images and added some more updates for ssf.txt. :man_shrugging:
\- LMK if I messed it up.

I imported ssf.txt into Excel, sorted by value, and added a column so I can keep track of what has been checked/changed and any problems loading games.
\- I'm using a Frankenstein'd ROMset so some games won't work.
\- Once the list of untested high-value games gets a bit longer, I'll probably post it in the BYOAC thread and ask for help.

The line that I deleted (`bbprot`) is an unplayable prototype.

So far, I've only found three games that I'm having trouble testing.
  `bosco` - Don't have the right ROM.
  `blitz` and `blitz99` - Don't have the CHDs.
